docs/2_3_fmu_distribution.adoc

```diff
@@ -208,3 +208,6 @@ _[New in FMI 2.0.4:]_
 ==== Terminals and Icons
 
 The optional directory `terminalsAndIcons` contains the definition of the Ports and Icons of the FMU as defined in the FMI 3.0 Specification, 2.4.9. Terminals and Icons.
+
```

Comment:
This clarification doesn't help much.

Reply:
As we will explain this in 3.0.1 and the Implementer's Guide: Do we need an additional explaination for FMI 2.0.4 at all or can we close this PR?